### PR TITLE
do not claim the deployer has cancelled/is cancelling the deploy

### PR DESCRIPTION
**Pull request by grosser**
... and other wording improvements

fixes https://github.com/zendesk/samson/issues/1880

### CODE REVIEW

File: app/models/deploy.rb [hunk 1]
```diff
@@ -42,16 +40,23 @@ def job_execution_queue_name
     "stage-#{stage.id}" unless stage.run_in_parallel
   end
 
-  def summary
-    "#{job.user.name} #{deploy_buddy} #{summary_action} #{short_reference} to #{stage.name}"
+  # job has almost identical code, keep it in sync
+  def summary(global: false)
```
Comment (jonmoter) [inline app/models/deploy.rb]: I don't understand what `global` means in this context.  Better name for variable?

File: app/models/deploy.rb [hunk 2]
```diff
@@ -42,16 +40,23 @@ def job_execution_queue_name
...
+    project_name = " #{project&.name}" if global
+    if ["cancelled", "cancelling"].include?(status)
+      "#{job.user.name}s deploy#{deploy_buddy} of #{short_reference} to#{project_name} #{stage&.name} is #{status}"
```
Comment (jonmoter) [inline app/models/deploy.rb]: Should have an apostrophe before the first "s"

**Comment by grosser** [comment]
we use `global` in a few places to mean `not scoped to the project` ...
when we want to show a deploy but it's not on the project page
... could change to `project_scoped: false` ... is that clearer ?

**Comment by jonmoter** [comment]
Okay, I get it.  So you're displaying the summary in a context that isn't specific to the project.

How about just `show_project_name: true`?

**Comment by grosser** [comment]
`show_project` then ... I'll update the other usages too ...

**Pull request merged**